src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Shapes/EllipsePage.xaml.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace SamplesApp.Windows_UI_Xaml_Shapes
+{
+	[SampleControlInfo("Shapes", "Ellipse")]
```

Comment:
It makes life easier if the sample name here exactly matches the name of the type (ie "EllipsePage")

Reply:
Same comment applies to other samples